README.md

```diff
@@ -54,3 +54,18 @@ or with the URL:
 ```bash
 https://grodecoder.streamlit.app/
 ```
+
+## Update the database
+
+Run the script to download coarse-grain model molecule data: 
+```bash
+python script/scrap_MAD.py
+```
+
+And run the script to download all-atom model molecule data: 
+```bash
+python script/scrap_charmm_gui_CSML.py
+```
+
+These scripts analyzes the database MAD (https://mad.ibcp.fr/explore) and CHARMM-GUI CSML (https://www.charmm-gui.org/?doc=archive&lib=lipid). It download the data - if it's not already exist. It extracts information that can be used to identify lipids in the GRO and PDB files and save it into a CSV file.  
```

Comment:
Could you please create subsections for both databases? And also mention explicitly the name of database file.

Reply:
Like that ? 
# Update database 
## CSML 
[Information/description]

## MAD
[information/description]

Reply:
yes